### PR TITLE
improved readme

### PR DESCRIPTION
**Pull request by nananananate**
wassup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,19 +6,63 @@ Basic Flask backend that has CRUD (Create, Read, Update, Delete) endpoints for a
...
+```
+
+**🐍 Create Python virtual environment**
+
+There are a good amount of depencies for this project -- it will be good practice to use a virtual environment, albeit not necessary.
+
+On macOS and Linux:
+
+```
+python3 -m virtualenv env
```
Comment (jogplus) [inline README.md:24]: I think this isnt right. It should be `python3 -m venv env`

File: README.md [hunk 2]
```diff
@@ -6,19 +6,63 @@ Basic Flask backend that has CRUD (Create, Read, Update, Delete) endpoints for a
...
+On Windows Command Line:
+
+```
+.\env\Scripts\activate.bat
+```
+
+One Windows Powershell
+
+```
+.\env\Scripts\activate.ps1
```
Comment (jogplus) [inline README.md:53]: Thank you for adding this!

File: README.md [hunk 3]
```diff
@@ -6,19 +6,63 @@ Basic Flask backend that has CRUD (Create, Read, Update, Delete) endpoints for a
...
+On macOS and Linux:
+
+```
+python3 -m virtualenv env
+```
+
+On Windows:
+
+```
+python -m venv env
```
Comment (jogplus) [inline README.md:30]: We might want to leave a comment to try `py -m venv env` as well? Since I saw that worked for someone on Windows

Comment (nananananate) [reply README.md:30]: should be added from the latest commit

File: README.md (hunk 1 above)
Comment (jogplus) [reply README.md:24]: bump

Comment (nananananate) [reply README.md:24]: mb

Comment (nananananate) [reply README.md]: should be resolved

**Pull request merged**